README.md

```diff
@@ -103,18 +103,36 @@ Please note that your package will be removed from the set if it is not kept up
 
 Adding a package is a manual process right now. We would like to add commands to make this process simpler, but for now, please follow these steps:
 
-- Tag a release of your library
-- Run the `dependencies` command to get the list of (transitive) dependencies, or copy the dependencies from your `psc-package.json` file
-- Make a pull request on the package set repository (against `master`) to add a new entry to `packages.json`. Use the dependency information above to fill in the fields, and the name of your new tag.
+- go to the [package-sets repository](https://github.com/purescript/psc-package) and fork the repository.
+- open the `packages.json` file and copy the existing scheme to add your package. The key will be the package name without the preceding `purescript-` as in Bower packages. It should have three fields defined:
```

Comment:
What does `scheme` refer to here?

Reply:
Oh I see, I had to read ahead a little. Maybe this could be something like "create a new entry for your package, based on the format of the existing packages"?

Reply:
ah, yeah. updated it to be clearer